### PR TITLE
docs: fix tls description and limitations

### PR DESCRIPTION
**Pull request by thesephirot**
fixes tls descriptino for knative and adds a general disclaimer about limitations

### CODE REVIEW

File: docs/docs/guides/installation/run/knative.mdx [hunk 1]
```diff
@@ -58,9 +56,7 @@ kn service create zitadel \
 --env ZITADEL_TLS_ENABLED=false \
 --env ZITADEL_EXTERNALDOMAIN=zitadel.default.127.0.0.1.sslip.io \
 --env ZITADEL_S3DEFAULTINSTANCE_CUSTOMDOMAIN=zitadel.default.127.0.0.1.sslip.io \
---arg "admin" --arg "start-from-init" --arg "--masterkey" --arg "MasterkeyNeedsToHave32Characters" \
---mount /tls.secret=secret:certs/tls.secret \
---mount /tls.key=secret:certs/tls.key
+--arg "admin" --arg "start-from-init" --arg "--masterkey" --arg "MasterkeyNeedsToHave32Characters" 
 ```
 
 or use the knative service yaml
```
[the remark this answers is not in the document]
Comment (eliobischof) [reply docs/docs/guides/installation/run/knative.mdx:62]: also add the whats next section
image

File: docs/docs/guides/installation/run/knative.mdx [hunk 2]
```diff
@@ -58,9 +56,7 @@ kn service create zitadel \
 --env ZITADEL_TLS_ENABLED=false \
 --env ZITADEL_EXTERNALDOMAIN=zitadel.default.127.0.0.1.sslip.io \
 --env ZITADEL_S3DEFAULTINSTANCE_CUSTOMDOMAIN=zitadel.default.127.0.0.1.sslip.io \
---arg "admin" --arg "start-from-init" --arg "--masterkey" --arg "MasterkeyNeedsToHave32Characters" \
---mount /tls.secret=secret:certs/tls.secret \
---mount /tls.key=secret:certs/tls.key
+--arg "admin" --arg "start-from-init" --arg "--masterkey" --arg "MasterkeyNeedsToHave32Characters"
```
[the remark this answers is not in the document]
Comment (eliobischof) [reply docs/docs/guides/installation/run/knative.mdx:59]: does it work without the --tlsMode disabled flag?

**Review by eliobischof — CHANGES_REQUESTED** [review]

File: docs/docs/guides/installation/run/knative.mdx [hunk 3]
```diff
@@ -1,6 +1,12 @@
 ## Disclaimer
 This guide is for development / demonstration purpose only and does NOT reflect a production setup.
 
+Things such as TLS termination and email verification will not be available unless you 
+
+- use an API gateway with valid certificates in front of the service
```
[the remark this answers is not in the document]
Comment (thesephirot) [reply docs/docs/guides/installation/run/knative.mdx:6]: nope, loadbalancing is heavily dependend on the clients setup

**Comment by thesephirot** [comment]
#notetomyself
cockroach db mem limits to 1GB

File: docs/docs/guides/installation/run/knative.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (thesephirot) [reply docs/docs/guides/installation/run/knative.mdx:59]: yes it does work without the flag

File: docs/docs/guides/installation/run/knative.mdx [hunk 4]
```diff
@@ -1,6 +1,12 @@
...
+- configure an appropriate email server
```
[the remark this answers is not in the document]
Comment (thesephirot) [reply docs/docs/guides/installation/run/knative.mdx:7]: done

File: docs/docs/guides/installation/run/knative.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (thesephirot) [reply docs/docs/guides/installation/run/knative.mdx:59]: removed admin

File: docs/docs/guides/installation/run/knative.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (thesephirot) [reply docs/docs/guides/installation/run/knative.mdx]: makes sense. I`ve added a link

**Pull request merged**